### DISCUSSION TITLE
放大文字文字偏左、全屏模式状态栏全黑

### DISCUSSION THREAD

**Discussion by linlin849** [plateaukao/einkbro#115] [Ideas]
放大文字后文字偏左，希望可以在中间
全屏模式时状态栏全黑没有隐藏也看不到时间
希望能支持自定义广告拦截，就是那种长按隐藏元素

**Comment by plateaukao** [comment]
@linlin849 
* 放大的功能是系統元件提供的，所以會歪掉就是會歪掉，在 EinkBro 做不了什麼來改善這情形。
* 請問你是用哪台設備？目前我只有在 OPPO 的機器上會有這問題。這比較是硬體廠商的問題。我在 Onyx BOOX 上的全螢幕都是正常的。
* 以後會考慮加入這類功能。(目前也有，只是時好時壞。你可以長按圖片，然後選右下角的 adblock，然後稍微編輯一下 url 試試，但有很大的機率會無效就是了)
